tests/test_fitAffineWcsTask.py

```diff
@@ -0,0 +1,274 @@
+# LSST Data Management System
+# Copyright 2008, 2009, 2010 LSST Corporation.
```

Comment:
Should copy this copyright blurb from a newer file (like one of the new modules you added).